daemon/unit_test.c

```diff
@@ -420,6 +420,7 @@ int unit_test_static_threads() {
 
     if (n < 2) {
         fprintf(stderr, "only %d static threads registered", n);
+        free(static_threads);
```

Comment:
Hi Vasili! Not a big, but any reason not to use `freez` ?

Reply:
Changed in latest commit.